### PR TITLE
WIP: Create release governance definition

### PR DESCRIPTION
**Pull request by kevinrizza**
Initial release governance doc intended to codify the project's release process and schedule

### CODE REVIEW

**Comment by cdjohnson** [comment]
Suggestion:  When we review these RFE's and don't yet accept them as valid, consider using a tag such as `triaged`, so we know we've started talking about it.

File: doc/contributors/releasegovernance.md
Comment (exdx) [inline doc/contributors/releasegovernance.md:17]: This seems like a lot of people to simply cut a minor release version, no? We're talking about 5 people at a minimum. If the number of approvers grows by 1, then a quorum would be 4 which means 6 people total. Seems like a lot of coordination for a go/no-go on a release. 

Do we really need so much review for the upstream community to cut a release? We want more frequent releases and for upstream members to be able to release on an accelerated schedule. I would think 2 `lgtms` and 1 `approve` would suffice? With the goal of the upstream community members being able to cut their own releases eventually with a different set of owners

File: doc/contributors/releasegovernance.md
Comment (exdx) [inline doc/contributors/releasegovernance.md:25]: What's the value of the 48 hour comment period besides just slowing down the release process once its been approved? I can understand sending out the email but cutting a new release seems like safe operation, not potentially contentious like an enhancement, is there really a freeze period required?

File: doc/contributors/releasegovernance.md
Comment (exdx) [inline doc/contributors/releasegovernance.md:39]: Maybe the upstream project can have more frequent releases? The kube release cycle mirrors the downstream OCP release cycle, which is fine, but having more frequent releases cut when new features make it into upstream could be another way to go

**Review by exdx — APPROVED** [review]
Thanks for putting this together @kevinrizza 

I think my thoughts differ on how much weight we need to put into an individual release and how much the core OLM folks need to be involved. I think we should let the upstream process go in its own direction and not apply a lot of policy to the release cycle. Let's discuss this more in our feature planning meeting.

File: doc/contributors/releasegovernance.md
Comment (ankitathomas) [reply doc/contributors/releasegovernance.md:25]: Will commenters be able to argue against cutting a release in the comment period? I think it makes sense to have some time for everyone to review the release before it goes out, but that I think that should be before the github issue gets approved.

File: doc/contributors/releasegovernance.md
Comment (timflannagan) [inline doc/contributors/releasegovernance.md:37]: Is this still accurate given the upstream/downstream split and we're no longer explicitly tied to the OCP release cadence?

Comment (timflannagan) [reply doc/contributors/releasegovernance.md:37]: (and I completely missed Dan's comment below that asks the same question)

**Comment by awgreene** [comment]
Hello,

This repository contains a number of PRs that have been open for an excessively long period of time, in some cases these PRs have been open for more than two years. In an effort to improve the health of our project, we are closing PRs that haven't been updated in the past 30 days or include features that we do not wish to adopt. This PR has been identified as a candidate meeting the requirements above and will be closed. If you believe that this PR should still be merged, please reopen the PR and explain why.

We will eventually release an official policy detailing our process for handling "stale" PRs and issues. If you would like to be involved in the conversation, please engage through [this Issue](https://github.com/operator-framework/rukpak/pull/553).

Thanks you,

Alex

**Pull request closed without merging**
